internal/testhelpers/queries/mother-query-X.groq

```diff
@@ -1,6 +1,7 @@
 //groq:version=0.x.x
 //groq:param:myParam="abc"
+//groq:param:baz="music"
```

Comment:
This shouldn't be needed? Or is it to check if it conflicts with the $baz param defined by the function? 🤔

Reply:
mmm 🤔  the spec says function parameters are regular parameters so I didn't do anything special regarding that